### PR TITLE
limits decimal points for SAM barplot marker

### PR DESCRIPTION
**Pull request by moontrip**
This will address https://github.com/VEuPathDB/web-monorepo/issues/760.

Screenshots are attached for before and after this PR. SCORE S. mansoni Cluster Randomized Trial is used.

# before
SAM bar decimal points1

# after
SAM bar decimal points2

### CODE REVIEW

File: packages/libs/components/src/plots/Barplot.tsx [hunk 1]
```diff
@@ -121,7 +121,12 @@ const Barplot = makePlotlyPlotComponent(
               orientation: orientation === 'vertical' ? 'v' : 'h',
               opacity: calculatedOpacity,
               type: 'bar',
-              text: showValues ? el.value : undefined,
+              // two decimal points for y values
+              text: showValues
+                ? el.value.map(
+                    (value: number) => `${Number.parseFloat(value.toFixed(2))}`
```
Comment (dmfalke) [inline packages/libs/components/src/plots/Barplot.tsx:127]: I think some unnecessary type conversions can be removed.

```suggestion
                    (value: number) => value.toFixed(2)
```

**Review by dmfalke — COMMENTED** [review]
@moontrip let me know what you think of my suggestion below.

File: packages/libs/components/src/plots/Barplot.tsx (hunk 1 above)
Comment (moontrip) [reply packages/libs/components/src/plots/Barplot.tsx:127]: Hi @dmfalke The reason why I introduced Number.parseFloat() is that I wanted to remove unnecessary zeros added to an integer value. For example, for the value of 10, 10.toFixed(2) becomes 10.00. I think it is better to show 10 only in that case?

Comment (dmfalke) [reply packages/libs/components/src/plots/Barplot.tsx:127]: Ah, that makes sense. Thanks for the explanation.

**Review by dmfalke — APPROVED** [review]

File: packages/libs/components/src/plots/Barplot.tsx (hunk 1 above)
Comment (moontrip) [reply packages/libs/components/src/plots/Barplot.tsx:127]: My pleasure 😃 Thank you for your review again 👍

**Pull request merged**